components/collective-navbar/index.js

```diff
@@ -313,17 +338,19 @@ const getDefaultCallsToActions = (collective, isAdmin, newNavbarFeatureFlag) =>
 };
 
 /**
- * Returns the main call to action that should be displayed as a button outside of the action menu.
- * This code could be factorized with `ActionsMenu.js`, as we want to have the same icons/actions/labels
- * here and there.
+ * Returns the main CTA that should be displayed as a button outside of the action menu in this component.
+ * Returns the second CTA that should be displayed as a button in ActionsMenu.js if only 2 CTAs.
  */
-const getMainAction = (collective, isAdmin, callsToAction) => {
+const getActions = (collective, isAdmin, callsToAction) => {
```

Comment:
Function name doesn't match its description. Since it returns the main call to action from `callsToAction`, so I think `getMainAction` was a better name